frontend/pages/DashboardPage/cards/ActivityFeed/components/ScriptDetailsModal/ScriptDetailsModal.tsx

```diff
@@ -29,60 +28,84 @@ const ScriptContent = ({ content }: IScriptContentProps) => {
   );
 };
 
+const StatusMessageRunning = () => (
+  <div className={`${baseClass}__status-message`}>
+    <p>
+      <Icon name="pending-partial" />
+      Script is running. To see if the script finished, close this modal and
+      open it again.
+    </p>
+  </div>
+);
+
+const StatusMessageSuccess = () => (
+  <div className={`${baseClass}__status-message`}>
+    <p>
+      <Icon name="success-partial" />
+      {`Exit code: 0 (Script ran successfully.)`}
+    </p>
+  </div>
+);
+
+const StatusMessageFailed = ({ exitCode }: { exitCode: number }) => (
+  <div className={`${baseClass}__status-message`}>
+    <p>
+      <Icon name="error-outline" />
+      {`Exit code: ${exitCode} (Script failed.)`}
```

Comment:
Could we take out the {``} here too and just put:

Exit code: {exitCode} (Script failed.)

Reply:
removed.